sml/include/sml/sml_shared.h

```diff
@@ -56,6 +56,43 @@ typedef int64_t i64;
 #define SML_TYPE_NUMBER_32 sizeof(u32)
 #define SML_TYPE_NUMBER_64 sizeof(u64)
 
+// A bitmap to configure workarounds.
+typedef int16_t sml_workarounds;
```

Comment:
would make this `uint`, so we can safely use all the bits.
i wonder if it should be 32 bits, this library is probably too large for 16 bit machines, otoh. we won't need that many entries anyway...